arbor/include/arbor/cable_cell.hpp

```diff
@@ -130,25 +130,24 @@ class cable_cell {
     // Density channels.
     void paint(const std::string& target, mechanism_desc);
 
+    // Properties.
+    void paint(const std::string& target, cable_cell_local_parameter_set);
+
```

Comment:
It wasn't implemented because it wasn't needed for the unit tests. I have added it, though I am still a little uncertain about providing multiple ways to paint on `cable_cell`s. 

Maybe it would be better to implement such flexibility into cell building tools that provide friendly interfaces for interacting with `sample_tree`s.

One useful workflow to support would be for users to interactively build a cell, then save it in a compact representation for later use (e.g. as input for a HPC simulation). We don't save information about what is painted where, which would facilitate this.

Reply:
In some sense, it's just one way of painting: a region name can be regarded itself as a region expression, once support is added for this to the region/locset code.

In terms of the compact representation, the design will definitely be driven by use case: there's the state of a discretized cell (which we'll need more or less for checkpointing anyhow) at the lower level, and the 3-d morphology and labelled subsections at a higher level of description; is there a case for an external representation of the pre-discretization 1-d cable cell?

Reply:
That is an interesting use case. It could be used to optimize both memory and time required during the model building phase (once we observe that this is a problem, of course).